index.js

```diff
@@ -4,15 +4,31 @@ var Promise = require('any-promise');
 var util = require('util');
 var format = util.format;
 
-function TimeoutError(message) {
+function TimeoutError(message, err) {
   Error.call(this);
   Error.captureStackTrace(this, TimeoutError);
   this.name = 'TimeoutError';
   this.message = message;
+  this.original = err;
```

Comment:
previous might be more accurate

Reply:
@mickhansen Was going for something similar to http://docs.sequelizejs.com/class/lib/errors/index.js~DatabaseError.html but I can change it if you want.

Reply:
I figured @contra and it makes sense when it's a wrapped error, in this case it's not a wrapped error but and independent error where errors might have occured in the past.

Reply:
@mickhansen Updated